styles/globals.css

```
@@ -0,0 +1,125 @@
+@import 'tailwindcss';
+@import 'tw-animate-css';
+
+@custom-variant dark (&:is(.dark *));
+
+:root {
+  --background: oklch(1 0 0);
```

Stylelint (17.2.0)

[error] Expected "0" to be "0deg" (hue-degree-notation)

(hue-degree-notation)